### ISSUE TITLE
cannot create a divider in the menu

### ISSUE THREAD

**Issue by hannigreen** [steviebaa/mui-nested-menu#10]
Hi,
first, I enjoy your package very much, thank you!
How can I create a divider between some elements in my context menu?
something similar to what MUI's menu have: 
![image](https://user-images.githubusercontent.com/107905728/174715677-3246f579-1c1b-4fcf-af0f-8a30b9dedb06.png)

**Comment by steviebaa** [comment]
Hi @hannigreen,

Thanks for raising this. It made me notice that the way of creating a context menu isn't very nice.

I've added a prop to the `ContextMenu` in version [2.1.0](https://www.npmjs.com/package/mui-nested-menu/v/2.1.0) called `menuItems` which will take an array of items. This should give you more flexibility. I've added a [demo to the docs](https://mui-nested-menu.vercel.app/context-menu).
